device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-D96C16/buffer_ports_t0.j2

```diff
@@ -0,0 +1,21 @@
+{%- macro generate_port_lists(PORT_ALL) %}
```

Comment:
This sku doesn't have pcbb changes. so the buffers_pool_sizes*.j2 and buffers_extra_queues.j2 don't exist